variables.tf

```diff
@@ -62,6 +62,12 @@ variable "backup_enabled" {
   default     = true
 }
 
+variable "backup_location" {
+  type        = string
+  description = "The region where the backup will be stored. If not specified, backups will be stored in the multiregion geographically nearest to the instance"
+  default     = ""
```

Comment:
You should either set this a specific location or remove this so a user knows that it is required

Reply:
I wanted to keep it backward compatible for all other places where its used (& this module is used at several places). If its left empty, it'll use default location - that is create backups in a multiregion that is geographically closest to the location where the clousql instance is created. Just wanted to have a functionality to set custom location as well.